themes/bootstrap3/theme.config.php

```diff
@@ -245,6 +246,7 @@
             'status-pending' => 'FontAwesome:clock-o',
             'status-ready' => 'FontAwesome:bell',
             'status-unavailable' => 'FontAwesome:times',
+            'status-unknown' => 'FontAwesome:circle',
```

Comment:
@crhallberg, why is this added here? Is this something that was accidentally omitted and needs to be restored? Is it the result of a merge error? It doesn't seem directly related to the rest of the PR, so I'm wondering what to do with it....

Reply:
It was something that came up as an important missing option. I can put it elsewhere but I think it was introduced in this update.

Reply:
I don't see any code that uses status-unknown in this PR... I do see code related to unknown statuses that uses the status-indicator icon, however. Is that a bug that needs correction, or am I missing something else? If that does need changing, I'd propose waiting until the rest of this work is completed and fixing that in a subsequent pass, just to avoid confusion.

Reply:
It is something I intended to use for easier customization of [this part of the code](https://github.com/vufind-org/vufind/pull/2447/files/0e93724cc7cb6eb8f1eca9e90a6900af9b86de93#diff-6702ea8e4bf5c523d3dede9b16eabf9c02fbfd62f714e242cf993ebe3f16d26aR63) but had not added yet. There's much about this part of the code that irks me. Why are we checking for location only if the status is unknown?, for example.

Reply:
I think the idea is that we only have a status when an item has a location. If it is available or unavailable, we can assume that the status has a location, since otherwise we wouldn't have been able to determine a binary status. When the status is unknown, it could either mean there is no location (in which case we don't know the status because the status is unknowable) or that something is wrong (we know where it is, but not what its status is). It's only useful to display the unknown status when there is an associated location. Does that make any sense? Maybe it's worth adding some code comments to clarify this.

Reply:
I think the code could be clearer by putting the location check as an early escape on the loop.

Reply:
I think the question is whether there are any scenarios where location is undefined but status is known. I don't _think_ that is likely, but I wouldn't swear to it without closer review.

Reply:
In any case, unless you feel very strongly otherwise, we should probably get this code settled as-is before introducing further refactoring.